lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1141,6 +1141,22 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'If `true`, disables Action Cable instrumentation.'
         },
+        :disable_action_mailbox => {
+          :default => false,
+          :public => true,
+          :type => Boolean,
+          :dynamic_name => true,
+          :allowed_from_server => false,
+          :description => 'If `true`, disables ActionMailbox instrumentation.'
+        },
+        :disable_action_mailer => {
```

Comment:
Does dependency detection automatically check these configs? I see that they are marked as dynamic and you aren't referencing them directly, but I can't remember how that works.

Reply:
Removed the "dynamic" key/val pair for the new configs with [354f0de](https://github.com/newrelic/newrelic-ruby-agent/pull/1740/commits/354f0de483791f52daec5bae941b6e513015e127)